src/test/java/tr/com/srdc/cda2fhir/jolt/EncounterActivityTest.java

```diff
@@ -143,6 +143,7 @@ private static void runTest(EncounterActivityGenerator generator, String caseNam
 		IEntryResult cda2FhirResult = rt.tEncounterActivity2Encounter(ec, new BundleInfo(rt));
 
 		Bundle bundle = cda2FhirResult.getBundle();
+		FHIRUtil.printJSON(bundle, "srs/test/resources/output/encounterBundle.json");
```

Comment:
There is an OUTPUT_PATH defined in this class. Please use that one.

Reply:
Yeah I need to remove this... I was just ad hoc testing something.